org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ClassFile.java

```diff
@@ -5764,6 +5764,17 @@ public byte[] getBytes() {
 		}
 		return this.bytes;
 	}
+
+	/**
+	 * EXTERNAL API
```

Comment:
what is the semantiv of EXTERNAL_API

Reply:
I believe this is just a hint for the maintainer, see `getBytes()`.